test/tsrc/validated_move_test.c

```diff
@@ -608,9 +622,10 @@ void test_validated_move_many() {
 }
 
 void test_validated_move() {
+  test_validated_move_distinct_kwg();
+  return;
```

Comment:
remove this early return you had for testing

Reply:
ah, thanks for catching, fixed that